src/utils/charts/candleChartSeries.ts

```diff
@@ -24,6 +24,12 @@ export function generateCandleSeries(
     },
     showSymbol: false,
   };
+  if (value.type === ChartType.scatter) {
+    sp['symbolSize'] = value.scatterSymbolSize;
```

Comment:
we'll need a fallback here.
`symbolSize` can't be undefined - which it is if you have an existing plot-config with scatter (checkout main, create a new plot config with a scatter plot - **close the page** (due to a HMR bug in pinia-persisted-state it seems)- checkout your branch and reopen the page - notice that the scatter plot doesn't render).